pep-0420.txt

```diff
@@ -5,6 +5,7 @@ Last-Modified: $Date$
 Author: Eric V. Smith <eric@trueblade.com>
 Status: Final
 Type: Standards Track
+Topic: Packaging
```

Comment:
I'd exclude this -- it's more about the behaviour of the import system, rather than a Packaging interoperability standard or so.